asreview/webapp/src/PublicationZone.js

```diff
@@ -101,6 +101,19 @@ const PublicationZone = (props) => {
               secondary={'Download a complete copy of this project. Ideal to share or import later on.'}
             />
           </ListItem>
+          <Divider component="li" />
+          <ListItem
+            button
+            onClick={props.finishProject}
+            alignItems="flex-start"
+            disabled={!props.showExportResult}
+            key="finish-project"
+          >
+            <ListItemText
+              primary={props.projectFinished ? "Mark as in review" : "Mark as finished"}
```

Comment:
"Mark as in review", what about "Mark as finished (undo)"?

Reply:
"Mark screening as finished"?

Reply:
> "Mark screening as finished"?

How about:
"Mark reviewing as finished"
"Mark reviewing as finished (undo)"

Reply:
I propose to use 'mark screening as complete' or '.. finished' and use 'project published' for when we implemented the link with OSF with the option to add a doi for the technical logfile plus a seperate doi for the paper.

Reply:
![MarkAsFinished](https://user-images.githubusercontent.com/17449217/90773112-c056cb80-e2f5-11ea-9562-244c434359fd.png)
![MarkAsFinishedUndo](https://user-images.githubusercontent.com/17449217/90773134-c6e54300-e2f5-11ea-91d9-775632480e03.png)